cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -1198,15 +1196,9 @@ subroutine rectgrid
       if (icepack_warnings_aborted()) call abort_ice(error_message=subname, &
          file=__FILE__, line=__LINE__)
 
-      !$OMP PARALLEL DO PRIVATE(iblk,i,j)
-      do iblk = 1, nblocks
-         do j = 1, ny_block
-         do i = 1, nx_block
-            ANGLE(i,j,iblk) = c0              ! "square with the world"
-         enddo
-         enddo
-      enddo
-      !$OMP END PARALLEL DO
+      hm (:,:,:) = c0
```

Comment:
Is it necessary to initialize to zero in all of the block (also valid for kmt and angle?

Reply:
There are a few different grid initialiizations (popgrid, popgrid_nc, latlon, rectangular, etc), and they don't all initialize the variables in the same way.  I believe zero-ing these out here is required for the rectangular grid option that is used for the box setups.  It is not needed for the popgrid option that we use for our more standard gx3+ testing.  It could be that a better solution is to make all the grid options behave in the same way, but it wasn't clear to me what those requirements should be, so I went this direction.